onnxruntime/core/framework/kernel_registry_manager.cc

```diff
@@ -23,7 +23,7 @@ Status KernelRegistryManager::CreateKernel(const onnxruntime::Node& node,
   {
     for (auto& registry : custom_kernel_registries_) {
       status = registry->TryCreateKernel(node, execution_provider, session_state.GetInitializedTensors(),
-                                         session_state.GetOrtValueNameIdxMap(), session_state.GetFuncMgr(), op_kernel);
+                                         session_state.GetOrtValueNameIdxMap(), session_state.GetFuncMgr(), session_state.GetDataTrasnferMgr(), op_kernel);
```

Comment:
Can you verify if DataTransferMgr is passed into subgraph session state too? I fixed a bug recently for GetFuncMgr() in creating subgraph session state. Besides, seems GetDataTrasnferMgr() is a typo.

Reply:
good point! verified. it's passed.